ddterm/com.github.amezin.ddterm.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=Drop Down Terminal
+NoDisplay=true
+Comment=Another drop down terminal extension for GNOME Shell. With tabs. Works on Wayland natively.
+Icon=terminal
+Exec=gdbus call --session --dest org.gnome.Shell --object-path /org/gnome/Shell/Extensions/ddterm --method com.github.amezin.ddterm.Extension.Activate
+Actions=
```

Comment:
Is there any reason to include empty `Actions=`? I. e. is the default non-empty somehow?

Reply:
I generated this file with a GUI tool, called [MenuLibre](https://flathub.org/apps/details/org.bluesabre.MenuLibre). It probably leaves it there. I thought it had some importance but now that I tested it without `Actions=` I see it doesn't. Line removed!